### PR TITLE
Move progressMonitor tool to Lexis

### PR DESCRIPTION
**Pull request by tribal-tec**

### CODE REVIEW

File: doc/Changelog.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 # git master
 
+* [157](https://github.com/HBPVIS/ZeroEQ/pull/157):
```
Comment (rdumusc) [inline doc/Changelog.md]: #161

**Comment by rdumusc** [comment]
+1 but just to be sure: no lexis subproject needed?

File: doc/Changelog.md (hunk 1 above)
Comment (tribal-tec) [inline doc/Changelog.md]: I corrected that and pushed. Github is lying! :)

**Comment by tribal-tec** [comment]
No, lexis is included/used in the applications which use ZeroEQ.

**Pull request merged**